esp/templates/survey/review.html

```diff
@@ -34,9 +83,10 @@ <h1>Survey Responses for {{ program.niceName }} </h1>
 {% if perclass_data %}
 {% for c in perclass_data %}
```

Comment:
Could we have each separate class on the results page filtered to a teacher have some sort of nice sections for the classes like either a border between them or boxes or something? It looks kinda plain with just the headings.